src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -23,6 +23,7 @@
         // Elements
 
         var startBankIdAppButtonElement = document.querySelector('#bankIdLoginStartApp');
+        var qrCodeElement = document.querySelector("#qr-code");
```

Comment:
I would like this not to be an ID, as ID:s can collide. Please give that image element a class of something like 'qr-code-image` and find it by `statusElement.querySelector(´qr-code-image´);`

Reply:
oki doki!